src/managers/GuildManager.js

```diff
@@ -128,23 +128,29 @@ class GuildManager extends BaseManager {
     return super.resolveID(guild);
   }
 
+  /**
+   * Options used to create a guild.
+   * @typedef {Object} GuildCreateOptions
+   * @property {number} [afkChannelID] The ID of the AFK channel
+   * @property {number} [afkTimeout] The AFK timeout in seconds
+   * @property {PartialChannelData[]} [channels=[]] The channels for this guild
```

Comment:
Just so I'm sure, is this defined?

Reply:
Yup, it's defined [here](https://github.com/discordjs/discord.js/blob/da6f21a3fca084f27d115754fed5fff8091cce77/src/managers/GuildManager.js#L74). Same, issue there tho, the id props are documented as `number`.